tests/framework/extensions/charts/charts.go

```diff
@@ -255,3 +354,30 @@ func GetChartCaseEndpoint(client *rancher.Client, host, path string, isWithTLS b
 		Body: bodyString,
 	}, nil
 }
+
+// GetProjectByName is a helper function that returns the project by name in a specific cluster
+func GetProjectByName(client *rancher.Client, clusterID, projectName string) (*management.Project, error) {
```

Comment:
I like the function I think this belongs in different package. Maybe extentsions/projects/projects.go